src/js/clickToCopy.js

```diff
@@ -0,0 +1,25 @@
+const removeAnchorClickedClass = function () {
+  const anchorLinks = document.querySelectorAll(
+    '.header-anchor.anchor-clicked',
+  );
+  anchorLinks.forEach((anchorLink) =>
+    anchorLink.classList.remove('anchor-clicked'),
+  );
+};
+
+const copyAnchorLink = function (anchorLink) {
+  navigator.clipboard.writeText(anchorLink.href).then(() => {
+    removeAnchorClickedClass();
+    anchorLink.classList.add('anchor-clicked');
+  });
+};
+
+const clickToCopy = function () {
+  const anchorLinks = document.querySelectorAll('.header-anchor');
+
+  anchorLinks.forEach((anchorLink) =>
+    anchorLink.addEventListener('click', () => copyAnchorLink(anchorLink)),
+  );
+};
+
+module.exports = clickToCopy;
```

Comment:
This is the first runtime JS we have on the site, so I think it'd be good to add tests (and test coverage) for it. I think that would involve a few changes to the `jest.config.js` -- or possibly separate config files for client-tests and src-tests or something. I'd be happy to do that or pair with you on it, but also happy if you want to go ahead and tackle it.

Reply:
Agree 100%! I'm happy to take a stab at it and reach out if I get stuck! 🙏🏼